mlflow/tracking/fluent.py

```diff
@@ -380,6 +380,17 @@ def start_run(
             # if `log_system_metrics` is not specified, we will check environment variable.
             log_system_metrics = MLFLOW_ENABLE_SYSTEM_METRICS_LOGGING.get()
         if log_system_metrics:
+            # Ensure psutil is installed. It was moved to an optional dependency, as it doesn't
+            # have binary for Arm64 Linux and requires build from CPython which was a headache.
+            if importlib.util.find_spec("psutil") is None:
+                raise MlflowException(
+                    "Failed to start system metrics monitoring as package `psutil` is not "
+                    "installed. Run `pip install psutil --upgrade` to resolve the issue, "
```

Comment:
```suggestion
                    "installed. Run `pip install psutil` to resolve the issue, "
```

Reply:
ah we don't need, just a random sugestion from copilot